test/demo-autoresearch/experiment.py

```
@@ -0,0 +1,68 @@
+
+    mse = sum(errors) / len(errors)
+    elapsed = time.time() - start
+
+    # Print in autoresearch-parseable format
+    print(f"predictions: {len(errors)}")
+    print(f"max_error:   {max(errors):.6f}")
+    print("---")
+    print(f"val_mse:          {mse:.6f}")
+    print(f"training_seconds: {elapsed:.1f}")
+    print(f"total_seconds:    {elapsed:.1f}")
+    print(f"peak_vram_mb:     0.0")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)